benchmark-scripts/format_avc_mp4.sh

```diff
@@ -70,14 +49,33 @@ then
 	exit 1
 fi
 
+result=${1/.mp4/"-$WIDTH-$FPS-bench.mp4"}
+if [ -f ../sample-media/$result ]
+then
+	echo "Skipping...conversion was already done for ../sample-media/$result."
+	exit 0
+fi
+
+if [ ! -f ../sample-media/$1 ] && [ ! -f ../sample-media/$result ]
+then	
+	wget -O ../sample-media/$1 $2
+fi
+
+if [ ! -f ../sample-media/$1 ]
+then
+	echo "ERROR: Can not find video file or 
+	"
+	show_help
+	exit 1
+fi
+
 
 if [ $HAS_FLEX_140 == 1 ] || [ $HAS_FLEX_170 == 1 ] || [ $HAS_ARC == 1 ]
 then
         TAG=sco-dgpu:2.0
 
 else
-        echo "ERROR: Requires Intel Flex/Arc GPU"
-	exit 1
+        TAG=sco-soc:2.0
 fi
```

Comment:
@brian-intel :   if either one of docker images can work, why do we need this part of logic at all? 
we could just use either one of images and then we don't even need the logic to detect GPU for downloading media file,  right?

Reply:
The main issue is most users will only build the image based on their platform. We could possibly change the logic to use whichever image is available on the system.

Reply:
yea that's true: so if we could do grep docker images then we can get rid of gpu detection logic then -- i like that idea